sympy/solvers/tests/test_solveset.py

```diff
@@ -709,22 +709,10 @@ def test_solveset_complex_tan():
 
 def test_solve_trig():
     from sympy.abc import n
-    assert solveset_real(sin(x), x) == \
-        Union(imageset(Lambda(n, 2*pi*n), S.Integers),
-              imageset(Lambda(n, 2*pi*n + pi), S.Integers))
-
     assert solveset_real(sin(x) - 1, x) == \
         imageset(Lambda(n, 2*pi*n + pi/2), S.Integers)
 
-    assert solveset_real(cos(x), x) == \
-        Union(imageset(Lambda(n, 2*pi*n - pi/2), S.Integers),
-              imageset(Lambda(n, 2*pi*n + pi/2), S.Integers))
-
-    assert solveset_real(sin(x) + cos(x), x) == \
-        Union(imageset(Lambda(n, 2*n*pi - pi/4), S.Integers),
-              imageset(Lambda(n, 2*n*pi + 3*pi/4), S.Integers))
-
-    assert solveset_real(sin(x)**2 + cos(x)**2, x) == S.EmptySet
+    assert solveset_real(sin(x)**2 + cos(x)**2, x) == S.EmptyS
```

Comment:
These are already [here](https://github.com/Shekharrajak/sympy/blob/master/sympy/solvers/tests/test_solveset.py#L749).
Now `solveset` returns desired output for these.which was previously @XFAIL. @asmeurer